website/static/js/logFeed.js

```diff
@@ -83,9 +83,13 @@ var Log = function(params) {
         return ret;
     });
 
-    //helper function to strip the leading slash for file or folder in log template
-    self.stripLeadingSlash = function(path){
-        return path.replace(/^\//, '');
+    //helper function to strip the slash for file or folder in log template
+    self.stripSlash = function(path, name){
```

Comment:
@chennan47, can you please make this change? After that, I think this should be good to go. :octopus: